systemSmartContractsConfig.toml

```diff
@@ -25,3 +26,14 @@
     MinPassThreshold = 300
     MinVetoThreshold = 50
     EnabledEpoch = 1000000
+
+[DelegationManagerSystemSCConfig]
+    BaseIssuingCost = "0" #0 eGLD
+    MinCreationDeposit = "1250000000000000000000" #1.25K eGLD
+    EnabledEpoch = 1000000 #enable epoch should not be 0
+
+[DelegationSystemSCConfig]
+    MinStakeAmount = "10000000000000000000" #10 eGLD
+    EnabledEpoch   = 1000000 #enable epoch should not be 0
```

Comment:
i think you can remove this comment. also from above

Reply:
will remain like this for now